sunbeam-python/sunbeam/commands/juju_utils.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import click
+from rich.console import Console
+from snaphelpers import Snap
+
+from sunbeam.commands.juju import RegisterRemoteJujuUserStep, UnregisterController
+from sunbeam.jobs.common import run_plan
+from sunbeam.jobs.deployment import Deployment
+
+LOG = logging.getLogger(__name__)
+console = Console()
+
+
+@click.command()
+@click.option(
+    "-f",
+    "--force",
+    is_flag=True,
+    help="Replace if controller exists with same name",
```

Comment:
"Force replacement if controller already exists with the same name"

Reply:
Fixed